### PR TITLE
Add German translation

### PR DESCRIPTION
**Pull request by Swarkin**
Adding German translation to GodSVG.

A couple things to note:
- I wasn't sure if i should change the word "Tag" to something else because I couldn't find a similar and short word for it in German. Problem is, "Tag" means "day" and could be confusing.
- Same applies to "Handles", couldn't find a similar word.
- Is the word SVG feminine, masculine or neutral in German? Which articles should be used? I assumed it to be masculine.
- All other translated strings should be good to go.

Please tell me whether those translations are alright (preferably by someone whose main language is German as well).

### CODE REVIEW

File: translations/translation_sheet.csv [hunk 1]
```diff
@@ -1,54 +1,52 @@
...
-#final_size,Final size,Краен размер
-#export_configuration,Export Configuration,Конфигурация на експорта
-#add_tag,Add new tag,Добави нов таг
+﻿,en,bg,de
+#remember_svg_setting,Remember SVG,Запомни SVG,SVG merken
+#remember_svg_desc,"When this box is checked, the SVG will persist between session.","Когато тази кутийка е отметната, SVG-то ще бъде запомнено между сесиите.","Wenn diese Option aktiviert ist, wird das SVG zwischen Sitzungen wiederhergestellt."
+#remember_window_mode_settings,Remember window mode,Запомни режима на прозореца,Fenstermodus wiederherstellen
+#remember_window_mode_desc,"When this box is checked, the window can remain restored between sessions.","Когато тази кутийка е отметната, прозорецът може да остане малък между сесиите.","Wenn diese Option aktiviert ist, wird das Fenster zwischen Sitzungen wiederhergestellt."
+#language,Language,Език,Sprache
+#open,Open,Отвори,Öffnen
```
Comment (Kiisu-Master) [inline translations/translation_sheet.csv:7]:
```suggestion
#invert_zoom,Invert zoom direction,Zoomrichtung umkehren
#invert_zoom_desc,Swaps zoom in and zoom out,Vertauscht Hinein- und Herauszoomen
#open,Open,Отвори,Öffnen
```

File: translations/translation_sheet.csv [hunk 2]
```diff
@@ -1,54 +1,54 @@
...
-#export_configuration,Export Configuration,Конфигурация на експорта
-#add_tag,Add new tag,Добави нов таг
+﻿,en,bg,de
+#remember_svg_setting,Remember SVG,Запомни SVG,SVG merken
+#remember_svg_desc,"When this box is checked, the SVG will persist between session.","Когато тази кутийка е отметната, SVG-то ще бъде запомнено между сесиите.","Wenn diese Option aktiviert ist, wird das SVG zwischen Sitzungen wiederhergestellt."
+#remember_window_mode_settings,Remember window mode,Запомни режима на прозореца,Fenstermodus wiederherstellen
+#remember_window_mode_desc,"When this box is checked, the window can remain restored between sessions.","Когато тази кутийка е отметната, прозорецът може да остане малък между сесиите.","Wenn diese Option aktiviert ist, wird das Fenster zwischen Sitzungen wiederhergestellt."
+#language,Language,Език,Sprache
+#invert_zoom,Invert zoom direction,Zoomrichtung umkehren
+#invert_zoom_desc,Swaps zoom in and zoom out,Vertauscht Hinein- und Herauszoomen
```
Comment (Kiisu-Master) [inline translations/translation_sheet.csv:8]: sorry, forgot about the missing bg translation
```suggestion
#invert_zoom,Invert zoom direction,,Zoomrichtung umkehren
#invert_zoom_desc,Swaps zoom in and zoom out,,Vertauscht Hinein- und Herauszoomen
```

Comment (Swarkin) [reply translations/translation_sheet.csv:8]: @MewPurPur do you wanna fill out the missing translations here?

Comment (MewPurPur) [reply translations/translation_sheet.csv:8]: Обърни посоката на увеличение

Would appreciate if the description is tweaked to say "Swaps zoom in and zoom out with the mouse wheel"
(Разменя увеличението и намалянето с колелцето на мишката)

**Comment by Alex2782** [comment]
> * I wasn't sure if i should change the word "Tag" to something else because I couldn't find a similar and short word for it in German. Problem is, "Tag" means "day" and could be confusing.

A few days ago I also wanted to start translating into German, I'm not sure about "Tag" either.
Maybe better "Element" or "Objekt" ?


inkscape, german docs:
https://inkscape.org/de/ueber/features/

or "Knoten"? (Node like in Godot)

**Comment by MewPurPur** [comment]
All these are valid names right now. Tags are a type of XML node called usually "element" in that context.

**Comment by Swarkin** [comment]
I replaced "Tag" with "Element" now, fixed a small grammar mistake and changed the translation of `#invert_zoom_desc` to [what Mew requested](https://github.com/MewPurPur/GodSVG/pull/171#discussion_r1381788941).

**Comment by MewPurPur** [comment]
I'm an Espeon, not a Mew actually. Anyway, thanks!

**Comment by Alex2782** [comment]
`Ungültige formatierung.` -> `Ungültige Formatierung.` the rest looks right at first glance. 👍

**Comment by MewPurPur** [comment]
OK, we can slip this in a PR localizing tabs.

**Pull request merged**